news.html

```diff
@@ -222,7 +222,7 @@ <h4>QuTiP is Presented in an Invited Talk at the RIKEN Berkeley Workshop on Quan
         <h4>QuTiP is Featured on Nature's Physics Blog</h4>
     </div>
     <div class="col-md-4" style='margin-top: 25px'>
-        <a href="http://blogs.nature.com/onyourwavelength/2019/01/09/the-rise-of-open-source-in-quantum-physics-research/"><p>The Rise of Open-Source in Quantum Physics Research</p></a>
+        <a href="https://blogs.nature.com/onyourwavelength/2019/01/09/the-rise-of-open-source-in-quantum-physics-research/"><p>The Rise of Open-Source in Quantum Physics Research</p></a>
```

Comment:
```suggestion
        <a href="http://blogs.nature.com/onyourwavelength/2019/01/09/the-rise-of-open-source-in-quantum-physics-research/"><p>The Rise of Open-Source in Quantum Physics Research</p></a>
```

Reply:
Oh, _mostly_ I checked the links as I changed them, but I didn't bother actually clicking through to Nature...  I assumed that they'd be fine.  Looking above, looks like I missed one or two as well.

Not shown in the diff are the other few http links I found that didn't seem safe to change.